pkg/resourceskeleton/testdata/asset-skeleton.yaml

```diff
@@ -26,8 +26,29 @@
 - resourceConfigId: AlloyDBInstance
 # not supported by asset inventory
 - resourceConfigId: AlloyDBUser
-# not supported by asset inventory
-- resourceConfigId: ArtifactRegistryRepository
+
+# Service: apikeys.googleapis.com
+- resourceConfigId: APIKeysKey
+  asset:
+    ancestors:
+    - projects/1234567890
+    - organizations/1234567890
+    name: //apikeys.googleapis.com/projects/1234567890/locations/global/keys/examplekey
+    asset_type: apikeys.googleapis.com/Key
+  expectedSkeleton:
+    apiVersion: apikeys.cnrm.cloud.google.com/v1alpha1
+    kind: APIKeysKey
+    metadata:
+      name: examplekey
+    spec:
+      projectRef:
+        external: "1234567890"
+
+# Service: artifactregistry.googleapis.com
+
+- resourceConfigId: ArtifactRegistryRepository # not supported by asset inventory
+
```

Comment:
Nit: You seem to have added a few newlines, which most of this yaml file doesn't have. Is there any best practice we should follow?

Reply:
I'm just trying to make it more human-readable (it's a human-generated/edited file I believe), so I put in white space and section headers to break up the text around my edit point.